modules/social_features/social_group/modules/social_group_request/config/install/message.template.approve_request_join_group.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - activity_logger
+third_party_settings:
+  activity_logger:
+    activity_bundle_entities:
+      group_content-group_content_type_7fcb76fdf61a9: group_content-group_content_type_7fcb76fdf61a9
+      group_content-group_content_type_c8bae2527294c: group_content-group_content_type_c8bae2527294c
+    activity_action: change_status_group_membership_request_entity_action
+    activity_context: approved_request_join_group_activity_context
+    activity_destinations:
+      notifications: notifications
+      email: email
+    activity_create_direct: true
+    activity_aggregate: false
+    activity_entity_condition: ''
+template: approve_request_join_group
+label: 'Approve request to join a group'
+description: 'My group membership request is approved'
+text:
```

Comment:
In the values below the `&nbsp;` is added because of the config export, could you remove that?

Reply:
Yep. Removed